modules/340-extended-monitoring/images/extended-monitoring-exporter/src/extended-monitoring.go

```diff
@@ -0,0 +1,232 @@
+package main
```

Comment:
you also need to rebuild the image.  https://github.com/Horiodino/deckhouse/blob/extended-monitoring-alerts-separately-%231311/modules/340-extended-monitoring/images/extended-monitoring-exporter/Dockerfile 
As an example, it is better to use this option https://github.com/Horiodino/deckhouse/blob/extended-monitoring-alerts-separately-%231311/modules/340-extended-monitoring/images/image-availability-exporter/Dockerfile